lib.typ

```diff
@@ -37,19 +37,19 @@
 
   set table.header(repeat: false)
 
-  show table.cell.where(y: 0): set text(weight: "bold", size: 12pt)
+  show table.cell.where(y: 0): set text(weight: "bold", size: 1.10em)
```

Comment:
Where did you get those values from? Why is it exactly 1.10em?

Reply:
The default font size is 11pt, so these em values retain the previous font sizes with the font size set to 11pt. This one should probably be 1.09

In this case, $\frac{12pt}{11pt} = 1.09 em$